Do not use nightly as default toolchain, use it locally as in start(-actix)

Using nightly as default might not be good for all environments, maybe switch to per project configuration. This template only compiles on nightly:

```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/server_fn_macro-0.2.5/src/lib.rs:1:46
  |
1 | #![cfg_attr(not(feature = "stable"), feature(proc_macro_span))]
  |  
```

- Add a local rust-toolchain.toml and change README.md as in https://github.com/leptos-rs/start for actix: https://github.com/leptos-rs/start/pull/19

--- a/README.md
+++ b/README.md
@@ -41,10 +41,9 @@ cargo leptos watch
 By default, `cargo-leptos` uses `nightly` Rust, `cargo-generate`, and `sass`. If you run into any trouble, you may need to install one or more of these tools.
 
 1. `rustup toolchain install nightly --allow-downgrade` - make sure you have Rust nightly
-2. `rustup default nightly` - setup nightly as default, or you can use rust-toolchain file later on
-3. `rustup target add wasm32-unknown-unknown` - add the ability to compile Rust to WebAssembly
-4. `cargo install cargo-generate` - install `cargo-generate` binary (should be installed automatically in future)
-5. `npm install -g sass` - install `dart-sass` (should be optional in future
+2. `rustup target add wasm32-unknown-unknown` - add the ability to compile Rust to WebAssembly
+3. `cargo install cargo-generate` - install `cargo-generate` binary (should be installed automatically in future)
+4. `npm install -g sass` - install `dart-sass` (should be optional in future
 
 ## Compiling for Release
 ```bash

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,0 +1,3 @@
+
+[toolchain]
+channel = "nightly"

